nearcore/src/config.rs

```diff
@@ -1773,9 +1773,13 @@ fn test_config_from_file() {
         // values is probably not worth it but there may be some other defaults
         // we want to ensure that they happen.
         let want_gc = if has_gc {
-            GCConfig { gc_blocks_limit: 42, gc_fork_clean_step: 420 }
+            GCConfig {
+                gc_blocks_limit: 42, gc_fork_clean_step: 420, num_epochs_to_keep_store_data: 5
```

Comment:
Could you also edit `../../testdata/example-config-gc.json` file and add `num_epochs_to_keep_store_data` there to make sure it’s properly read?

Reply:
Just did.